inc/saihostintf.h

```diff
@@ -186,7 +191,7 @@ typedef enum _sai_hostif_trap_type_t
     /** default action is forward */
     SAI_HOSTIF_TRAP_TYPE_IGMP_TYPE_V3_REPORT = 0x000000009,
```

Comment:
put SAI_HOSTIF_TRAP_TYPE_START and SAI_HOSTIF_TRAP_TYPE_END Macros, like other objects.